mods/d2k/maps/shellmap/map.yaml

```diff
@@ -33,102 +33,1571 @@ Players:
+		SkipMakeAnims:
+	Actor390: spicebloom
+		Location: 103,14
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor391: spicebloom
+		Location: 117,17
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor392: spicebloom
+		Location: 8,11
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor393: spicebloom
+		Location: 12,23
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor394: spicebloom
+		Location: 9,42
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor395: spicebloom
+		Location: 20,49
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor396: spicebloom
+		Location: 8,64
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor397: spicebloom
+		Location: 11,80
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor398: spicebloom
+		Location: 16,69
+		Owner: Neutral
+		SkipMakeAnims:
+	Actor399: silo
+		Location: 24,92
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor405: power
+		Location: 27,96
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor404: silo
+		Location: 24,93
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor403: silo
+		Location: 25,93
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor402: silo
+		Location: 26,93
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor401: silo
+		Location: 26,92
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor400: silo
+		Location: 25,92
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor406: wall
+		Location: 24,91
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor407: wall
+		Location: 25,91
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor408: wall
+		Location: 26,91
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor409: wall
+		Location: 27,91
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor410: wall
+		Location: 27,92
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor411: wall
+		Location: 27,93
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor412: wall
+		Location: 27,94
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor413: wall
+		Location: 26,94
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor414: wall
+		Location: 25,94
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor415: wall
+		Location: 24,94
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor416: wall
+		Location: 23,94
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor417: wall
+		Location: 23,93
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor418: wall
+		Location: 23,92
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor419: wall
+		Location: 23,91
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor420: refinery
+		Location: 5,87
+		Owner: Atreides
+		SkipMakeAnims:
+	Actor421: barracks
+		Location: 18,89
 		Owner: Atreides
+		SkipMakeAnims:
+	Actor422: refinery
+		Location: 25,19
+		Owner: Harkonnen
+		SkipMakeAnims:
+	Actor423: refinery
+		Location: 89,20
+		Owner: Harkonnen
+		SkipMakeAnims:
+	Actor424: guntower
+		Location: 94,19
+		Owner: Harkonnen
+		SkipMakeAnims:
+	Actor425: rockettower
+		Location: 84,23
+		Owner: Harkonnen
+		SkipMakeAnims:
 
 Smudges:
 
 Rules:
+	Player:
+		-ConquestVictoryConditions:
 	World:
 		-CrateSpawner:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		ResourceType@Spice:
 			ValuePerUnit: 0
-		WormManager:
-			Minimum: 1
-			Maximum: 1
+		LoadWidgetAtGameStart:
+			Widget: MAINMENU
```

Comment:
You don't need this anymore. We removed it during the preparation of the in-game map editor. https://github.com/OpenRA/OpenRA/pull/7217